libs/sdk-ui-charts/styles/scss/headline.scss

```diff
@@ -1,6 +1,7 @@
 // (C) 2007-2018 GoodData Corporation
-@import "~@gooddata/goodstrap/lib/core/styles/typo-mixins";
-@import "~@gooddata/goodstrap/lib/core/styles/layout";
+@import "~@gooddata/sdk-ui-kit/styles/scss/variables";
```

Comment:
Not 100% sure about this, but maybe we can loose the variables import, as it is imported inside typo-mixins below. I'm no sass expert so I don't know if it makes any difference..